code/array/array-new-vs-fixnum-times-map.rb

```diff
@@ -0,0 +1,17 @@
+require "benchmark/ips"
+
+ELEMENTS = 9
```

Comment:
@nateberkopec, @deniskorobicyn, @JuanitoFatas: Thanks for the kind words. :smiley: I added the graph to the PR description, and linked to the PR from the `README`.